### DISCUSSION TITLE
Secutity issue - Logs and Conf files includes cleartext passwords and keys!

### DISCUSSION THREAD

**Discussion by Clouseau68** [rustdesk/rustdesk#1644] [General]
All passwords and keys are shown as cleartext! This may never ever be possible!

Folder:
Config
C:\Users\%User\AppData\Roaming\RustDesk\config
RustDesk.toml
RustDesk2.toml
Logs
C:\Users\%User\AppData\Roaming\RustDesk\log
All log files!

Also you can see your password in the client. That's not good either. If you need to add permannet password it should be differ that this temprorary password. Need to be able to add seperate password for unattanded acces.

**Comment by blaloc7** [comment]
I am also wondering about this. This is a security risk an needs to be changed!

**Comment by martinJomt** [comment]
in debian linux 12 the same:  
cat /var/log/rustdesk-server/hbbr.log
cat //var/log/rustdesk/signalserver.log

& clientside

**Reply by open-trade** [reply]
For hbbs/hbbr, it does not have password, yes, the public key is in cleartext, but why public can not be cleartext, it is `public`. :(

**Reply by martinJomt** [reply]
Thanx!

**Comment by open-trade** [comment]
Already fixed in client side two years ago
